wadm/lib/wadm/deployments/deployment_monitor.ex

```diff
@@ -195,9 +200,26 @@ defmodule Wadm.Deployments.DeploymentMonitor do
   @impl true
   def handle_info({:lattice_changed, lattice, _event}, state) do
     case state.recon_state do
-      # in backoff/cooldown mode, incoming events have no effect but will trigger
-      # a check after cooling off
+      # in backoff/cooldown mode, incoming events will only trigger link definitions
+      # as they are idempotent and can be applied multiple times (in the worst case)
+      # without side effects.
+      # Other events have no effect but will trigger a check after cooling off
       :reconciling ->
+        case actions_to_reconcile(state.spec, lattice) do
+          {cmds, _skip, _err} ->
+            case Enum.filter(cmds, fn %Wadm.Reconciler.Command{cmd: cmd} ->
+                   cmd == :put_linkdef
+                 end) do
+              [] ->
```

Comment:
Does `do_reconcile` work with an empty list of commands? If so removing this `case` would make this code more readable